api/v3/MessageTemplate.php

```diff
@@ -80,6 +80,17 @@ function civicrm_api3_message_template_delete($params) {
 function _civicrm_api3_message_template_get_spec(&$params) {
   // fetch active records by default
   $params['is_active']['api.default'] = 1;
+  $params['is_reserved']['api.default'] = 0;
```

Comment:
@mattwire doing some local testing it appears the addition of this line is breaking the syntax conformance test.

Reply:
@seamuslee001  Ok, I've removed that line, let's see if it works now